src/loggingFunctions.js

```diff
@@ -0,0 +1,33 @@
+var logUtil = require('./logUtil');

```

Comment:
this replaces all of our logging functions and builds them all using the same code.Makes changing behavior pretty simple.